Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -11,10 +11,135 @@
 #include <AMReX_MultiFab.H>
 
 
+/**
+* \brief Get the value of arr in the neighbor (i_n, j_n) on the plane with normal 'dim'.
+*        I.E. If dim==0 it return arr(i, j + i_n, k + j_n),
```

Comment:
Separating title from body of the docs with an empty newline like this:
```suggestion
*
*        I.E. If dim==0 it return arr(i, j + i_n, k + j_n),
```